GAE/test/org/akvo/flow/xml/FlowXmlObjectReaderTests.java

```diff
@@ -125,6 +130,17 @@ void testParseValidForm() throws IOException {
         assertEquals("Den enda frågan", q.getTranslationMap().get("sv").getText());
     }
 
+    @Test
+    void testParseGrouplessForm() throws IOException {
+
+        SurveyDto testFormDto = PublishedForm.parse(GROUPLESS_XML_FORM, true).toDto(); //be strict
+        assertNotEquals(null, testFormDto);
+
+        assertEquals("Foo", testFormDto.getName());
+        assertEquals("1.0", testFormDto.getVersion());
+        assertEquals(null, testFormDto.getQuestionGroupList());
+    }
+
```

Comment:
Please create a separate issue for the `TODO` for adding more tests thats below. 👍